modules/katamari/src/test/ts/atomic/api/Regex/RegexTest.ts

```diff
@@ -0,0 +1,17 @@
+import { describe } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+import * as Regex from 'ephox/katamari/api/Regex';
+
+describe('Regex.escapeRegExp', () => {
+  assert.equal('abcd\\.', Regex.escapeRegExp('abcd.'), 'period, asterisc',);
```

Comment:
There's additional commas here at the end that shouldn't be there either (after the message). This should be failing with linter errors I believe.

Reply:
Also, all these assertions. are back to front. It goes `equal(actual, expected, message)`, whereas you've used `equal(expected, actual, message)`.

- [ ] Fix assertions

Reply:
updated